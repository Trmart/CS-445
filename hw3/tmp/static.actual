.   .   .   Child: 0  Const 111 [line: 4]
.   .   .   Child: 0  Id: x [line: 6]
.   .   .   Child: 0  Id: x [line: 7]
.   .   .   Child: 0  Id: x [line: 8]
.   .   Child: 0  Var: x of static type int [line: 4]
.   .   Child: 1  Call: output [line: 6]
.   Child: 1  Compound [line: 3]
.   .   .   Child: 1  Const 222 [line: 7]
ERROR(6): Symbol 'output' is not declared.
ERROR(8): Symbol 'output' is not declared.
ERROR(LINKER): A function named 'main()' must be defined.
Func: ( returns type void [line: 2]
Number of errors: 3
Number of warnings: 0
.   .   Sibling: 1  Assign: <= [line: 7]
.   .   Sibling: 2  Call: output [line: 8]
