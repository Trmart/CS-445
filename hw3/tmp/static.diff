0a1,8
> .   .   .   Child: 0  Const 111 [line: 4]
> .   .   .   Child: 0  Id: x [line: 6]
> .   .   .   Child: 0  Id: x [line: 7]
> .   .   .   Child: 0  Id: x [line: 8]
> .   .   Child: 0  Var: x of static type int [line: 4]
> .   .   Child: 1  Call: output [line: 6]
> .   Child: 1  Compound [line: 3]
> .   .   .   Child: 1  Const 222 [line: 7]
3c11,13
< Number of errors: 2
---
> ERROR(LINKER): A function named 'main()' must be defined.
> Func: ( returns type void [line: 2]
> Number of errors: 3
4a15,16
> .   .   Sibling: 1  Assign: <= [line: 7]
> .   .   Sibling: 2  Call: output [line: 8]
