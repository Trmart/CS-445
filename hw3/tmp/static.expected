ERROR(6): Symbol 'output' is not declared.
ERROR(8): Symbol 'output' is not declared.
Number of errors: 2
Number of warnings: 0
